src/flippy.nim

```diff
@@ -684,7 +683,25 @@ proc rotate*(image: Image, angle: float): Image =
     beta = sin(radians)
   if alpha == 0.0 and beta == 0.0:
     return
-  result = result.shearX(alpha).shearY(beta).shearX(alpha).trim()
+  let
+    newWidth = int(abs(float(image.width) * sin(radians)) +
+                   abs(float(image.height) * cos(radians)))
+    newHeight = int(abs(float(image.width) * cos(radians)) +
+                    abs(float(image.height) * sin(radians)))
+    sheared = image.shearX(alpha).shearY(beta).shearX(alpha)
+    widthOffset = (sheared.width - newWidth) div 2
+    heightOffset = (sheared.height - newHeight) div 2
+
+  result = newImage(
```

Comment:
I think you want to use get sub image rather then implement it again: https://github.com/treeform/flippy/commit/0c1a9f911d7ce83785e944387d92d3924339c377#diff-1a8532401e3ca61cb035330df0955bf8L646

Reply:
Ah yeah much better that way